lib/rubocop/cop/gusto/explicit_time_units.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Gusto
+      class ExplicitTimeUnits < Base
+        MSG = "Avoid adding/subtracting integers or floats directly to Date/Time/DateTime. " \
+          "Use explicit time methods instead (e.g., `.days`, `.hours`)."
+
+        RESTRICT_ON_SEND = %i(+ - << >>).freeze
+
+        # Allowed time unit methods (both singular and plural)
+        TIME_UNIT_METHODS = %i(
```

Comment:
It would be simpler not to enumrate these and just accept any method call on a number as satisfying the rule